GPU/GLES/ShaderManagerGLES.cpp

```diff
@@ -586,6 +587,31 @@ void LinkedShader::UpdateUniforms(u32 vertType, const VShaderID &vsid) {
 	if (dirty & DIRTY_STENCILREPLACEVALUE) {
 		glUniform1f(u_stencilReplaceValue, (float)gstate.getStencilTestRef() * (1.0f / 255.0f));
 	}
+
+	if ((dirty & DIRTY_TEXLOD) && gstate_c.Supports(GPU_SUPPORTS_EXPLICIT_LOD) && u_texlod) {
```

Comment:
Shouldn't uniformMask handle the u_texlod (and ideally `GPU_SUPPORTS_EXPLICIT_LOD`)?  Also, isn't -1 the invalid value?

-[Unknown]